<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceDetailDialog</class>
 <widget class="QDialog" name="DeviceDetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>장치 상세 정보</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_device_info">
     <property name="text">
      <string>장치 정보 출력 예정</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="table_sensors">
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string>센서명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>유형</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>값</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>시간</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="button_toggle_status">
       <property name="text">
        <string>상태 변경</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_close">
       <property name="text">
        <string>닫기</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
